CHANGELOG.md

```diff
@@ -27,3 +27,18 @@
 # 0.3.0
 - Feature: An optional `timeout` parameter can be provided to the `config` object passed to `Scrapegoat` to indicate the number of milliseconds to wait for the server to send the response before aborting the request.
 
+# 0.4.0
+- fix(parser): convert dates to ISO-Strings
```

Comment:
Could you stick to the previous format (no angular commit message convention)? Since we're not using [`standard-version`](https://github.com/conventional-changelog/standard-version) here yet it makes no sense to use that convention.